src/__tests__/downshift.aria.js

```diff
@@ -1,12 +1,14 @@
 import React from 'react'
-import {render as renderToDOM} from 'react-testing-library'
+import {render, cleanup} from 'react-testing-library'
 import Downshift from '../'
 import {resetIdCounter} from '../utils'
 
 beforeEach(() => {
   resetIdCounter()
 })
 
+afterEach(cleanup)
```

Comment:
Rather than having an `afterEach(cleanup)` in each file, let's do: `import 'react-testing-library/cleanup-after-each'` at the top of each file. :+1:

Thanks!

Reply:
Yea sounds good 👍  Don't know why it says I dismissed your review after I squashed the commits tho 😅